lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -2633,9 +2633,29 @@ static Value NearestInterpolate(OpBuilder &b, Location loc,
     Value outFP = b.create<arith::SIToFPOp>(loc, b.getF32Type(), outInt);
     Value proj = b.create<arith::DivFOp>(loc, outFP, scale);
 
+    Value nearestFP;
     // get nearest pixel using floor
-    Value nearestFP = b.create<math::FloorOp>(loc, proj);
-
+    if (nearestMode == "floor") {
```

Comment:
this should still include the case `nearestMode = ""` since this is default behavior.